pkg/services/serviceaccounts/manager/service.go

```diff
@@ -39,6 +39,27 @@ func ProvideServiceAccountsService(
 	}
 	serviceaccountsAPI := api.NewServiceAccountsAPI(s, ac, routeRegister)
 	serviceaccountsAPI.RegisterAPIEndpoints(cfg)
+
+	basicKeys := store.GetNonServiceAccountAPIKeys(context.Background())
```

Comment:
Can we do this in the background rather than on start up?

Reply:
I wasn't sure about the user experience here.  There won't be much change, but it could lead to situations where someone looks at a menu, and when they reload/update, it changes without their input.

Also not sure if the performance drag on the database is better in one burst or spread out during user interaction.